src/Tizen.NUI.Components/Controls/Control.cs

```diff
@@ -55,6 +55,14 @@ static Control()
             ThemeManager.AddPackageTheme(DefaultThemeCreator.Instance);
         }
 
+        /// <summary>
+        /// This is used to improve theme performance.
+        /// </summary>
+        static public void PreLoad()
```

Comment:
did you intend to make only this API visible in the IntelliSense?
otherwise, please add "[EditorBrowsable(EditorBrowsableState.Never)]" without <since_tizen> tag.

Reply:
Done

Reply:
Done